src/cephadm/cephadm

```diff
@@ -3678,6 +3671,16 @@ def prepare_ssh(
             logger.info('Deploying %s service with default placement...' % t)
             cli(['orch', 'apply', t])
 
+def enable_cephadm_mgr_module(
```

Comment:
```
flake8 run-test: commands[0] | flake8 --config=tox.ini cephadm
cephadm:3687:1: E302 expected 2 blank lines, found 1
cephadm:3698:1: E302 expected 2 blank lines, found 1
2     E302 expected 2 blank lines, found 1
ERROR: InvocationError for command /home/jenkins-build/build/workspace/ceph-pull-requests/src/cephadm/.tox/flake8/bin/flake8 --config=tox.ini cephadm (exited with code 1)
```

please add two empty lines instead one before and after this function. you could run
```sh
tox -e flake8
```
under `src/cephadm` to verify your fix.

Reply:
Thank you @tchaikov for the help. I'm new to ceph and not fully aware of the coding practices we follow here. If there's any issue next time, I'll know where to look.

Reply:
no problem. thank you for your contribution!